docs/api/browser-window.md

```diff
@@ -1864,41 +1868,62 @@ machine has a touch bar and is running on macOS 10.12.1+.
 **Note:** The TouchBar API is currently experimental and may change or be
 removed in future Electron releases.
 
-#### `win.setBrowserView(browserView)` _Experimental_
+#### `win.setBrowserView(browserView)` _Experimental_ _Deprecated_
 
 * `browserView` [BrowserView](browser-view.md) | null - Attach `browserView` to `win`.
 If there are other `BrowserView`s attached, they will be removed from
 this window.
 
-#### `win.getBrowserView()` _Experimental_
+> **Note**
+> The `BrowserView` class is deprecated, and replaced by the new
+> [`WebContentsView`](web-contents-view.md) class.
```

Comment:
Yeah, we should put something in Breaking Changes. However, the deprecation of `BrowserView` isn't itself breaking—the only thing that's breaking about this change is the change to autoresize behavior on macOS.

I'm conflicted about whether to list the BrowserView deprecation in Breaking Changes, even though we do list other deprecations in that document, because I feel like there's not really much pressure to ever actually remove BrowserView. The wrapper around WebContentsView is fairly small and maintainable, and I don't feel like it's a huge burden to carry the old API forward more or less indefinitely. So it'll never really break, right?

Reply:
Makes sense. BrowserView being non-native code does make maintainability super easy 🤞